tutor/src/screens/grading-templates/editors.js

```diff
@@ -244,6 +246,8 @@ class TemplateForm extends React.Component {
 
   renderForm = (form) => {
     const { body } = this.props;
+    const namePlaceholder = this.props.template.task_plan_type == 'reading' ?
```

Comment:
It looks good! I think `template` can be added in the line above: `{ body, template }`.

Reply:
Good call! Added.